R/calculate_ilisi.R

```diff
@@ -0,0 +1,61 @@
+#' Calculate iLISI (integration Local Inverse Simpson's Index) scores
+#'
+#'
+#' @param integrated_sce The integrated SCE object
```

Comment:
Given the changes that are in #191, let's use `merged_sce` throughout. These could be either unintegrated and merged objects or objects with integration. 
```suggestion
#' @param merged_sce The merged SCE object containing data from multiple batches
```